core/solr_index.py

```diff
@@ -40,8 +40,13 @@ def _solr_escape(value):
     """
     return ESCAPE_CHARS_RE.sub(r'\\\g<char>', value)
 
-def _sort_facets_asc(solr_facets, field):
-    raw = solr_facets.get('facet_fields')[field]
+def _sorted_facet_counts(solr_counts, field):
+    """
+    Convert the raw solr facet data (counts, ranges, etc.) from a flat array
+    into a two-dimensional list sorted by the number of hits.  The result will
+    look something like this: (('field1', count1), 'field2', count2, ...)
```

Comment:
should this actually be
`(('field1', count1), ('field2', count2))` ?

I'm not super familiar with python syntax but the parentheses differences are confusing me if this is intentional, so can you give me a rundown if this is how the results are intended to look?

Reply:
I mean, I *did* say the results would look "something like" that.  `(('field1', count1), ('field2', count2))` does look quite a bit like what I wrote.

Reply:
(fixed)